Documentation/ExtensionArchitecture/ComponentsOfTYPO3Extension/MakingDataPersistable.rst

```diff
@@ -0,0 +1,19 @@
+.. _making-data-persistable:
+
+===================================
+Making Data Persistable by Extbase
+===================================
+
+Create an Entity Class and Repository
+-------------------------------------
+
+* Add `./Classes/Domain/Model/{entity name}.php`
+  * Extend `TYPO3\CMS\Extbase\DomainObject\AbstractEntity`
+  * Add database columns as class properties
```

Comment:
I think an example for this using the dummy properties defined in ext_tables.sql would help a lot here for clarity?